examples/nlp/token_classification/conf/ner_config.yaml

```diff
@@ -0,0 +1,86 @@
+# Named Entity Recoginition with pretrained BERT models
+
+pl:
+  trainer:
+    gpus: 1 # the number of gpus, 0 for CPU
+    num_nodes: 1
+    max_epochs: 3
+    max_steps: null # precedence over max_epochs
+    accumulate_grad_batches: 1 # accumulates grads every k batches
+    amp_level: O0 # O1/O2 for mixed precision
+    distributed_backend: dpp
+
+model:
+  data_dir: ??? # /path/to/data
+  class_balancing: null # choose from [null, weighted_loss]. Weighted_loss enables the weighted class balancing of the loss, may be used for handling unbalanced classes
+
+  train_ds:
+    prefix: train
+    max_seq_length: 128
+    pad_label: 'O'
+    num_samples: -1
+    ignore_extra_tokens: false
+    ignore_start_end: false
+    overwrite_processed_files: false
+    shuffle: true
+    batch_size: 2
```

Comment:
Isn't default batch size too low?

Reply:
it's just for debugging purposes, will increase